packages/server/modules/gatekeeper/services/trialEmails.ts

```diff
@@ -127,4 +129,8 @@ export const sendWorkspaceTrialExpiresEmailFactory =
       })
     )
     await Promise.all(sendEmailParams.map((params) => sendEmail(params)))
+    await mp.track('Workspace Trial Expiration Email Sent', {
+      workspaceId: args.workspace.id,
```

Comment:
Maybe add both `workspaceId` and `workspace_id` until we have a final decision on wether we can to move to camelCase or not? At least it will be grouped for now